docs/swagger.json

```diff
@@ -965,43 +984,41 @@
         "responses": {}
       }
     },
-    "/pinning/pins/{id}": {
-      "post": {
-        "description": "This endpoint replaces a pinned object.",
+    "/pinning/pins/{requestid}": {
+      "get": {
+        "description": "This endpoint returns a pinned object.",
```

Comment:
Not quite right. The object might not be pinned (aka a failed pin). Maybe "This endpoint returns a pin request"